### ISSUE TITLE
php artisan theme:widget mdui default     

### ISSUE THREAD

**Issue by Teeoo** [FaCuZ/laravel-theme#3]
Illuminate\Contracts\Filesystem\FileNotFoundException  : File does not exist at path 

  at /Users/teeoo/Desktop/Teeoo/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php:41
    37|         if ($this->isFile($path)) {
    38|             return $lock ? $this->sharedGet($path) : file_get_contents($path);
    39|         }
    40| 
  > 41|         throw new FileNotFoundException("File does not exist at path {$path}");
    42|     }
    43| 
    44|     /**
    45|      * Get contents of a file with shared access.

  Exception trace:

  1   Illuminate\Filesystem\Filesystem::get()
      /Users/teeoo/Desktop/Teeoo/vendor/facuz/laravel-themes/src/Commands/WidgetGeneratorCommand.php:191

  2   Facuz\Theme\Commands\WidgetGeneratorCommand::getTemplate("widgetClass")
      /Users/teeoo/Desktop/Teeoo/vendor/facuz/laravel-themes/src/Commands/WidgetGeneratorCommand.php:79

  Please use the argument -v to see more details.

**Comment by FaCuZ** [comment]
Thank! I fixed it: 3a93821312bc4048c561861f758fd61ea5cbd3da

**Comment by Teeoo** [comment]
That's the problem again.

> php artisan theme:widget demo default

```
Illuminate\Contracts\Filesystem\FileNotFoundException  : File does not exist at path
at /Users/teeoo/Desktop/TeeBlog/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php:41
    37|         if ($this->isFile($path)) {
    38|             return $lock ? $this->sharedGet($path) : file_get_contents($path);
    39|         }
    40| 
  > 41|         throw new FileNotFoundException("File does not exist at path {$path}");
    42|     }
    43| 
    44|     /**
    45|      * Get contents of a file with shared access.

 Exception trace:
```

```
  1   Illuminate\Filesystem\Filesystem::get()
      /Users/teeoo/Desktop/TeeBlog/vendor/facuz/laravel-themes/src/Commands/WidgetGeneratorCommand.php:192

  2   Facuz\Theme\Commands\WidgetGeneratorCommand::getTemplate("widgetClass")
      /Users/teeoo/Desktop/TeeBlog/vendor/facuz/laravel-themes/src/Commands/WidgetGeneratorCommand.php:79
```
